src/main/java/collab/FileManager.java

```diff
@@ -0,0 +1,31 @@
+package collab;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileManager {
+    public List<String> loadFile(String filePath) throws IOException {
+        ArrayList<String> stringList = new ArrayList<String>();
+        BufferedReader br = new BufferedReader(new FileReader(filePath));
+        try{
+            String str;
+            while ((str = br.readLine()) != null) {
+                stringList.add(str);
+            }
+        }catch(Exception e){
+            throw e;
+        }finally {
+            br.close();
+        }
```

Comment:
이 부분은 향후에 try ~ resources를 이용해서 finally를 제거하는 것이 좋을 것 같습니다.
참고: [https://codechacha.com/ko/java-try-with-resources/](https://codechacha.com/ko/java-try-with-resources/)


```suggestion
        
        try(BufferedReader br = new BufferedReader(new FileReader(filePath)) {
            String str;
            while ((str = br.readLine()) != null) {
                stringList.add(str);
            }
        }catch(Exception e){
            throw e;
        }
```